src/gui/qgsmapcanvas.cpp

```diff
@@ -1246,6 +1246,17 @@ void QgsMapCanvas::updateDevicePixelFromScreen()
   refresh();
 }
 
+void QgsMapCanvas::onElevationShadingRendererChanged()
```

Comment:
@vcloarec , would be nice for this to be added to src/quickgui/qgsquickmapcanvasmap.cpp too.

Reply:
Sorry, but I don't have experience with qt quick, and I can't test it properly. And this PR is already quit big...